pkg/tsdb/prometheus/healthcheck.go

```diff
@@ -57,13 +57,16 @@ func (s *Service) CheckHealth(ctx context.Context, req *backend.CheckHealthReque
 }
 
 func healthcheck(ctx context.Context, req *backend.CheckHealthRequest, i *instance) (*backend.CheckHealthResult, error) {
+
+	exprString := "1+1"
+
 	qm := models.QueryModel{
 		LegendFormat: "",
 		UtcOffsetSec: 0,
 		PrometheusDataQuery: dataquery.PrometheusDataQuery{
-			Expr:    "1+1",
+			Expr:    &exprString,
 			Instant: kindsys.Ptr(true),
-			RefId:   refID,
+			//RefId:   refID,
```

Comment:
Why is RefId no longer defined?

Reply:
Since we inherit (I am not sure if that's the correct term in golang) `RefId` from `DataQuery` struct this needs to be defined like the following:
```go
PrometheusDataQuery: dataquery.PrometheusDataQuery{
	Expr:    kindsys.Ptr("1+1"),
	Instant: kindsys.Ptr(true),
	DataQuery: dataquery.DataQuery{
		RefId: "A",
	},
},
```

Reply:
It's called embedding. See: https://eli.thegreenplace.net/2020/embedding-in-go-part-1-structs-in-structs/